### ISSUE TITLE
Quite unusable

### ISSUE THREAD

**Issue by ptoulza** [ayufan-rock64/android-7.1#17]
Hello Kamil, 

Thanks for your great work on atv 7.1 for Rock64.
I just got a Rock64 chip with 4gb RAM, and burnt your 0.3.13 android image on a class10 SDcard with Etcher.
First boot is a bit long (~5min).
- Then, directly on the launcher I get system errors (close or wait app).
- I had to launch the play store 3 times to be displayed the form activity to set my google account (had to tell the system to wait several times for the activity to respond).
- The netspeed for downloading is incredibly slow (120kB/s) on a gigabit link.
- Had to wait for 2 minutes for youtube to launch, but when launched, the speed is correct and I succeeded to play a streamed 1080p video.
- Next boot is about 2 minutes.

My feeling for now : The system is incredibly slow, and I wonder why it is.
It doesn't seem to be wakelocks, doesn't seem to be the RAM amount (3100MB free) either, maybe the class10 card ? 
There seems to be an input problem : google voice for input locks often, which leads to keyboard input not responding either (using android TV remote on smartphone).

I also tried the regular rock 64 android build and have the same impression of slowlyness. 
Do you have any opinion of this ?
Have you succeeded in having a smooth system your side (EMMC or microSD) ?
Thank you in advance and good luck !

**Comment by ptoulza** [comment]
I use this microSD : https://www.pearl.fr/article/CM642/carte-micro-sdhc-transflash-class-10-16-go

It's a mainstream 16GB class10.
Never tried with emmc. You think it could increase perf significantly ?

**Comment by droidforever** [comment]
I have the same type of problems.  Not so much the problem with speed but constant freeze.  Launcher errors I can very seldom even get into the android desktop.  I thought it was a bad board so I replaced it and still the same problems.  It always seems to freeze while loading recommendation.  I have used eMMC a Samsung Evo and a Samsung Evo+ all with the same result.  I would really love to use your rom as I love the way it looks but I have not been able to put it to the test because of all of the errors.  I have tried your last stable build and all of the pre-builds since all with the same result.  Your help is appreciated.
